lib/grape/endpoint.rb

```diff
@@ -276,6 +276,7 @@ def run
       end
     end
 
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Let's not do that and use `rubocop -a ; rubocop --auto-gen-config`. The second one will fix it. We don't want to litter the code with rubocop ignores, mostly using it for auto-formatting.

Reply:
Tried that at first, but it produced all kinds of unrelated changes. The correct command is: 
`bundle exec rubocop -a ; bundle exec rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 5000`
and really there should be a CI job that I can look at, that tells me that, but it seems you have to manually run CI checks on each commit? If you want more contributors, you should make this easier.

Reply:
I tend to ignore the unrelated changes, we just use rubocop for auto-formatting and minimal linting, so the rest is whatever.

GitHub doesn't let first time contributors auto-run CI on PRs for some security reason. I'll look into turning that off. FYI you can easily turn on GHA on your fork if you need it. Also, we do have 414 contributors, and sincerely appreciate your help!

Reply:
> GitHub doesn't let first time contributors auto-run CI on PRs for some security reason. I'll look into turning that off. 

That'd help a lot I think

> FYI you can easily turn on GHA on your fork if you need it. 

🤦🏼 I should have thought of that. 

Anyhow, thanks for merging!

Reply:
I switched it to the first option, but it doesn't let you disable this behavior. You can read about this more here: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

<img width="852" alt="Screenshot 2023-07-04 at 12 42 55 PM" src="https://github.com/ruby-grape/grape/assets/542335/7c4d7894-6f03-47f2-9a3d-5b1457bb88bd">